wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationController.swift

```diff
@@ -59,12 +61,14 @@ final class ConversationCreationController: UIViewController {
         return section
     }()
 
-    private lazy var optionsSections = [
-        guestsSection,
-        servicesSection,
-        receiptsSection,
-        shouldIncludeEncryptionProtocolSection ? encryptionProtocolSection : nil
-    ].compactMap { $0 }
+    private var optionsSections: [ConversationCreateSectionController] {
```

Comment:
It's not a **lazy** anymore because we need to update this array if the protocol has changed.